finality-aleph/src/sync/handler.rs

```diff
@@ -257,6 +273,8 @@ impl<I: PeerId, J: Justification, CS: ChainStatus<J>, V: Verifier<J>, F: Finaliz
     }
 
     /// Handle a single justification.
+    /// Return `Some(id)` if the highest known justification has just changed to
+    /// the one identified by `id`, unless it's also just been finalized.
```

Comment:
If you change as suggested above this comment also requires a similar change.

Reply:
fixed